rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -378,11 +401,21 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
                 counterparty=CPT_COWSWAP,
                 airdrop_identifier=airdrop_identifier,
             ):
+                if claim_has_payment:
+                    event.event_type = HistoryEventType.TRADE
+                    event.event_subtype = HistoryEventSubType.RECEIVE
+                    in_event = event
                 break
 
         else:
             log.error(f'Could not find the normal COW token claim for {self.evm_inquirer.chain_name} transaction {context.transaction.tx_hash.hex()}')  # noqa: E501
 
+        if claim_has_payment:
```

Comment:
to prevent errors check that `None not in (out_event, in_event)`

Reply:
This is not needed. If either is None then `maybe_reshuffle_events` will do nothing. But since yet it is an error I would log an error instead here.